providers/zendesksupport/parse.go

```diff
@@ -49,3 +51,11 @@ func getNextRecordsURL(node *ajson.Node) (string, error) {
 
 	return jsonquery.New(node).StrWithDefault("after_url", "")
 }
+
+func makeGetRecords(moduleID common.ModuleID, objectName string) common.NodeRecordsFunc {
```

Comment:
I understand the reasoning behind `make` as a prefix, but can we please just call this `getRecords` to make it easier to read?